File: B/Python/Disk.py
class Disk():

	type = "" 
	capacity = 0 
	price = 0 
	
	def __init__(self, type = "", capacity = 0, price = 0):
		self.type = type
		self.capacity = capacity
		self.price = price

	# methods set and get for price attribute
	def setType(self, type):
		self.type = type
	
	def getType(self):
		return self.type

	# methods set and get for capacity attribute
	def setCapacity(self, capacity):
		self.capacity = capacity

	def getCapacity(self):
		return self.capacity

	# methods set and get for price attribute
	def setPrice(self, price):
		self.price = price

	def getPrice(self):
		return self.price

File: B/Java/Disk.java
public class Disk
{
	private String type;
	private int capacity; 
	private int price; 

	public Disk(String type, int capacity, int price)
	{
		this.type = type;
		this.capacity = capacity;
		this.price = price;
	}

	public Disk()
	{

	}

	// methods set for type, capacity, and price attribute.
	public void setType(String type)
	{
		this.type = type;
	}
	public void setCapacity(int capacity)
	{
		this.capacity = capacity;
	}
	public void setPrice(int price)
	{
		this.price = price;
	}

	// methods get for type, capacity, and price attribute.
	public String getType()
	{
		return this.type;
	}
	public int getCapacity()
	{
		return this.capacity;
	}
	public int getPrice()
	{
		return this.price;
	}

}